src/schedule/2021-01-21-get-access-with-aaron-cannon.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Get Access with Aaron Cannon
+type: Episode Party
+date: 2021-01-28
+layout: layouts/event.html
+---
+
+Live listen and discussion of Aaron Cannon's [React Podcast](https://reactpodcast.com) episode on accessibility.  
+This episode aired July 11th, 2019 but remains relevant to React developers today.
+
+## Episode Summary
+
+This week, we talk accessibility pitfalls with Aaron Canon.
+
+Aaron is the co-founder and chief accessibility engineer at Accessible360 — where he uses his experience as a blind developer to improve real-world accessibility for all citizens of the web.
+
+He shares his first-hand experience on which practices work, which ones are bogus, where to focus our accessibility efforts, and which libraries provide the best starting point.
+
+I learned a ton. You will too.
+
+<iframe height="200px" width="100%" frameborder="no" scrolling="no" seamless src="https://player.simplecast.com/3366f2b7-1e86-441b-b03e-1abc874cca11?dark=false"></iframe>
```

Comment:
I _love_ the idea of embedding the podcast episode into the event page!

I bet you we could add the `src` to our frontmatter, and then in our layouts, we could conditionally add the `iframe` if it's an `Episode Party` event and it has an episode `src`.

Thoughts?

Reply:
i think that's a great idea! would probably prefer to resolve that in a follow-up, maybe even during our template filters sesh

Reply:
I dig it! I think there's a great opportunity for talking both about filters and the data cascade, and the merits of using either to populate content like this.